app/src/main/java/waiter/Routes.java

```diff
@@ -0,0 +1,42 @@
+package waiter;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Routes {
+
+    public Map<String, Map<String, Map<String, String>>> routes;
+
+    public Routes() {
+        this.routes = new HashMap<>();
+    }
+
+    private void addRoute(Route route) {
+        this.routes.putIfAbsent(route.getKey(), route.getValue());
+        if(this.routes.containsKey(route.getKey())) {
+            this.routes.get(route.getKey()).putAll(route.getValue());
+        }
+    }
+
+    public Routes constructRoutes() {
+        RouteDirector routeDirector = new RouteDirector();
+        RouteBuilder routeBuilder = new RouteBuilder();
+
+        this.addRoute(routeDirector.constructGetSimpleGet(routeBuilder));
+        this.addRoute(routeDirector.constructHeadSimpleGet(routeBuilder));
+        this.addRoute(routeDirector.constructGetSimpleGetWithBody(routeBuilder));
```

Comment:
If I want to add a route say `/foo` do I have to update this class?

Reply:
You would have to add an `addRoute` call to add the route `/foo`

Reply:
We now add routes at the top level (for the user), and we also no longer need to add a test for every route, addressed in [this commit](https://github.com/jonmanahan/waiter/pull/18/commits/3bc8754c8fee547caf992d65eb1c11005cecf743)